frontend/src/components/cluster-dashboard/cluster-graph-view/ClusterGraphView.tsx

```diff
@@ -0,0 +1,233 @@
+    styles.secondaryThemeColor +
+    '">' +
+    " Node</span>" +
+    "</div>" +
+    "</foreignObject>" +
+    "</svg>";
+  const nodeUrl =
+    "data:image/svg+xml;charset=utf-8," + encodeURIComponent(nodeSvg);
+  const [graphData, setGraphData] = useState({ nodes: [], edges: [] });
+
+  const options = {
+    nodes: {
+      shape: "circularImage",
+      borderWidth: 0.5,
+      borderWidthSelected: 1,
+      color: {
+        border: "transparent",
+        background: styles.primaryThemeColor,
+        highlight: {
+          border: styles.primaryThemeColor,
+          background: styles.primaryThemeColorLighter
+        },
+        hover: {
+          border: styles.primaryThemeColor,
+          background: styles.primaryThemeColorLighter
+        }
+      },
+      font: {
+        color: styles.primaryColor
+      },
+      shadow: false
+    },
+    edges: {
+      color: {
+        color: styles.secondaryColor,
+        hover: styles.primaryColor,
+        highlight: styles.primaryColor,
+        opacity: 0.8
+      },
+      width: 2,
+      selectionWidth: 0.5,
+      hoverWidth: 0.5
+    },
+    interaction: {
+      hover: true,
+      navigationButtons: true
+    },
+    physics: {
+      barnesHut: {
+        avoidOverlap: 0.2
+      },
+      minVelocity: 0.75
+    }
+  };
+
+  const loadGraphData = (): void => {
+    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const frameGraphData = (clusterData: Cluster): any => {
+      const leader = {
+        shadow: {
+          enabled: true,
+          color: styles.leaderNodeColor,
+          size: 15,
+          x: 1,
+          y: 1
+        }
+      };
+      const oldest = {
+        borderWidth: 3,
+        borderWidthSelected: 0,
+        color: {
+          border: styles.oldestNodeColor,
+          highlight: { border: styles.oldestNodeColor },
+          hover: { border: styles.oldestNodeColor }
+        },
+        shapeProperties: { borderDashes: [10, 10] }
+      };
+      //eslint-disable-next-line @typescript-eslint/no-explicit-any
+      const graph: any = {
+        nodes: [
+          {
+            id: 0,
+            label: "Akka Cluster",
+            image: clusterUrl,
+            size: 40,
+            borderWidth: 1,
+            borderWidthSelected: 2,
+            color: {
+              border: styles.primaryColor,
+              background: styles.secondaryColor,
+              highlight: {
+                border: styles.primaryColor,
+                background: styles.secondaryColorLighter
+              },
+              hover: {
+                border: styles.primaryColor,
+                background: styles.secondaryColorLighter
+              }
+            }
+          }
+        ],
+        edges: []
+      };
+
+      if (!clusterData || !clusterData.members) {
+        return graph;
+      }
+
+      clusterData.members.forEach((member, index) => {
+        const memberTitle = (
+          <React.Fragment>
```

Comment:
Nit: You can import `Fragment` along with useState and useEffect
```suggestion
          <Fragment>
```